lib/redshift_ecto/connection.ex

```diff
@@ -546,15 +546,15 @@ if Code.ensure_loaded?(Postgrex) do
     defp create_names(prefix, sources, pos, limit) when pos < limit do
       current =
         case elem(sources, pos) do
```

Comment:
Very strange one. As `case` should work, it should match `{table, schema, _alias}` pattern even if there is `:fragment` in the beginning. That somehow worked in the original code, and had been broken after dependencies updating.